spec/ParseUser.spec.js

```diff
@@ -2300,6 +2300,66 @@ describe('Parse.User testing', () => {
     });
   });
 
+  it('case insensitive login with username', async () => {
+    let user = new Parse.User();
+    user.set('username', 'test1');
+    user.set('password', 'test');
+    await user.signUp();
+    await Parse.User.logOut();
+    user = new Parse.User();
+    user.set('username', 'TEST1');
+    user.set('password', 'test');
+    await user.logIn();
```

Comment:
Should be wrapped in expectAsync so we know it was successful. Also all others below.

Reply:
Changed to expectAsync.